.evergreen/connect.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+LIB_DIR=${LIB_DIR:-"lib"}
+BUILD_TYPE=${BUILD_TYPE:-"Debug"}
+
+print_usage_and_exit () {
+    echo "Error: $1"
+    cat << EOF
+    connect.sh runs the connect example with a MongoDB URI.
+
+    Example usage:
+    MONGOC_INSTALL_PREFIX=/Users/kevin.albertson/install/mongo-c-driver-1.18.0 \\
```

Comment:
Up to you if you want to leave your name in here, or if it should be scrubbed. I am fine with it as-is.

Reply:
Leaving as-is.